app/views/layouts/_tracking_scripts.html.erb

```diff
@@ -1,184 +1,191 @@
+  </script>
+  <!-- End Visual Website Optimizer Asynchronous Code -->
 
-<!--
-Only show Drift on Signals page, otherwise show Usersnap bug report;
-this is because they both occupy the bottom right hand side of the screen,
-and there should be no bugs to report on the Signals pages so we want to
-encourage users to convert without distracting them with other options.
--->
-<% if request.original_fullpath.start_with? '/signals' %>
-<!-- Start of Async Drift Code -->
-<script>
-!function() {
-  var t = window.driftt = window.drift = window.driftt || [];
-  if (!t.init) {
-    if (t.invoked) return void (window.console && console.error && console.error("Drift snippet included twice."));
-    t.invoked = !0, t.methods = [ "identify", "config", "track", "reset", "debug", "show", "ping", "page", "hide", "off", "on" ],
-    t.factory = function(e) {
-      return function() {
-        var n = Array.prototype.slice.call(arguments);
-        return n.unshift(e), t.push(n), t;
-      };
-    }, t.methods.forEach(function(e) {
-      t[e] = t.factory(e);
-    }), t.load = function(t) {
-      var e = 3e5, n = Math.ceil(new Date() / e) * e, o = document.createElement("script");
-      o.type = "text/javascript", o.async = !0, o.crossorigin = "anonymous", o.src = "https://js.driftt.com/include/" + n + "/" + t + ".js";
-      var i = document.getElementsByTagName("script")[0];
-      i.parentNode.insertBefore(o, i);
-    };
-  }
-}();
-drift.SNIPPET_VERSION = '0.3.1';
-drift.load('rib3n3nefgnd');
-</script>
-<!-- End of Async Drift Code -->
-<% else %>
-<!-- UserSnap -->
-<script type="text/javascript">
-(function() { var s = document.createElement("script"); s.type = "text/javascript"; s.async = true; s.src = '//api.usersnap.com/load/566ea3b0-5719-4a7c-999f-b7f6604930c2.js';
-var x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x); })();
-</script>
-<% end %>
+  <!--
+  Only show Mouseflow on landing pages, otherwise the recording quota
```

Comment:
This might be better as a rails comment. Since these html comments will get rendered to page source